WeakAurasOptions/OptionsFrames/Update.lua

```diff
@@ -855,6 +855,17 @@ local function BuildUidMap(data, children, type)
     self.rootParent = parentId
   end
 
+  uidMap.Dump = function(self, uid)
+    if uid == nil then
+      uid = self:GetRootUID()
+    end
+    print(self:GetIdFor(uid))
```

Comment:
probably don't want that

Reply:
I did, it's debugging code for next time. The function "dumps" some information.

Reply:
Ah makes sense, I did not read it properly